cmd/manager/main.go

```diff
@@ -188,6 +195,34 @@ func main() {
 
 }
 
+func loadConfig() (*configuration.Registry, error) {
+	var configFilePath string
+	flag.StringVar(&configFilePath, "config", "", "path to the config file to read (if none is given, defaults will be used)")
+
+	// Override default -config switch with environment variable only if -config
+	// switch was not explicitly given via the command line.
+	configSwitchIsSet := false
+	flag.Visit(func(f *flag.Flag) {
+		if f.Name == "config" {
+			configSwitchIsSet = true
+		}
+	})
+	if !configSwitchIsSet {
+		if envConfigPath, ok := os.LookupEnv(configuration.HostEnvPrefix + "_CONFIG_FILE_PATH"); ok {
+			configFilePath = envConfigPath
+		}
+	}
+
+	return configuration.New(configFilePath)
+}
+
+func printConfig(cfg *configuration.Registry) {
+	log.Info("Registration Service configuration variables:")
+	for key, value := range cfg.GetAllRegistrationServiceParameters() {
+		log.Info(fmt.Sprintf("%s: %s", key, value))
```

Comment:
Separate log entries looks nice in terminal but will be harder to trace in Kibana. Wouldn't it better to keep all params in the one entry as separate key/value entries?

Reply:
Sure, I can do it. But please, don't want me to make the build failing again to show you the output :-).

Reply:
done in a115dff 
the output looks like:
```
{"level":"info","ts":1579192543.9154654,"logger":"cmd","msg":"Registration Service configuration variables:","key":"REGISTRATION_SERVICE_AUTH_CLIENT_CONFIG_RAW","value":"{ \"realm\": \"toolchain-public\", \"auth-server-url\": \"https://sso.prod-preview.openshift.io/auth\", \"ssl-required\": \"none\", \"resource\": \"crt\", \"clientId\": \"crt\", \"public-client\": true }","key":"REGISTRATION_SERVICE_ENVIRONMENT","value":"dev","key":"REGISTRATION_SERVICE_IMAGE","value":"quay.io/matousjobanek/registration-service:1579191974","key":"REGISTRATION_SERVICE_SERVICE_PORT_8080","value":"80","key":"REGISTRATION_SERVICE_SERVICE_HOST","value":"172.30.235.206","key":"REGISTRATION_SERVICE_PORT_80_TCP_PROTO","value":"tcp","key":"REGISTRATION_SERVICE_AUTH_CLIENT_LIBRARY_URL","value":"https://sso.prod-preview.openshift.io/auth/js/keycloak.js","key":"REGISTRATION_SERVICE_PORT","value":"tcp://172.30.235.206:80","key":"REGISTRATION_SERVICE_PORT_80_TCP","value":"tcp://172.30.235.206:80","key":"REGISTRATION_SERVICE_AUTH_CLIENT_PUBLIC_KEYS_URL","value":"https://sso.prod-preview.openshift.io/auth/realms/toolchain-public/protocol/openid-connect/certs","key":"REGISTRATION_SERVICE_REPLICAS","value":"1","key":"REGISTRATION_SERVICE_SERVICE_PORT","value":"80","key":"REGISTRATION_SERVICE_PORT_80_TCP_ADDR","value":"172.30.235.206","key":"REGISTRATION_SERVICE_PORT_80_TCP_PORT","value":"80"}
```